smart_contracts.rst

```diff
@@ -760,6 +760,13 @@ Also, the amount that a participant can receive cannot be bigger than the total
 
 ``(6 R)`` is deduced from ``(5 R)``. It is needed by the Raiden client in order to not allow a participant to :ref:`withdraw <withdraw-channel>` more tokens from the on-chain channel deposit than he is entitled to.
 
+::
+
+    (7 R) T2 + L2 - T1 - L1 <= D2
+
+``T2 + L2 - T1 - L1`` is the netted total transferred amount from ``P2`` to ``P1``. This amount cannot be bigger than ``P2``'s deposit -> a participant cannot transfer more tokens than what he has in the channel, during the lifecycle of a channel.
+This ``MUST`` be ensured by the Raiden client.
```

Comment:
So, I think `(7 R) T2 + L2 - T1 - L1 <= D2` is actually what the Raiden client `MUST` enforce.
However, the Raiden client only enforces `(5 R) AB1 = D1 - W1 + T2 - T1 - L1; AB1 >= 0, AB1 <= TAD` as far as I know.
So, we might want to prove that `(5 R)` ensures `(7 R)`